src/features/login/loginSlice.js

```diff
@@ -0,0 +1,60 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from '../../axios'
+
+import ls from 'local-storage'
+const initialState = {
+  value: 0,
+  loggedIn:false,
+  status: 'idle',
+  error:''
+};
+
+export const createTokenAsync = createAsyncThunk(
+  'user/fetchToken',
+  async (data,thunkAPI) => {
+    try {
+        //const response = await fetch(`url`); //where you want to fetch data
+        //Your Axios code part.
+        const response = await axios.post('/api-token-auth/',data)
+        ls.set('token', response.data.token)
+        return response.data.token
+  
+    } 
+      catch (error) {
+          alert("Authentication denied")
```

Comment:
Use react-toastify instead of showing general alerts.

Reply:
Kunal will add